testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java

```diff
@@ -97,6 +98,7 @@
  * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
+@EnableFeature(Profile.Feature.ADMIN_FINE_GRAINED_AUTHZ)
```

Comment:
Do we really need to enable this feature?

Reply:
Hi Pedro! Yes, it's needed, if not some tests fail, if I remember correctly several social providers test the token exchange ([github among them](https://github.com/keycloak/keycloak/blob/d6f50351f09b5bdce893c39d5c29b2bf7f5647c9/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java#L376)). That's why I know we don't launch these tests too often. :smile: